### PR TITLE
qemu_storage: added a resize method

### PR DESCRIPTION
**Pull request by hereischen**
1. added qemu-img resize wrapper
2. fixed some style issues

id: 1641497
Signed-off-by: Haotong Chen <hachen@redhat.com>

### CODE REVIEW

**Comment by hereischen** [comment]
@YongxueHong  @lolyu , please help review, thanks.

**Comment by hereischen** [comment]
@luckyh please help review， thanks

File: virttest/qemu_storage.py [hunk 1]
```diff
@@ -631,6 +634,23 @@ def amend(self, params, cache_mode=None, ignore_status=False):
...
         return cmd_result
 
+    def resize(self, size, shrink=False, ignore_status=False):
+        """
+        Qemu image resize wrapper.
+
+        :param size: string of size representations.(eg. +1G, -1k, 1T)
+        :param shrink: boolean
+        :param ignore_status: whether to raise an exception when command
+                              returns =! 0 (False), or not (True)
```
Comment (vivianQizhu) [inline virttest/qemu_storage.py]: `=!/!=`?

File: virttest/qemu_storage.py [hunk 2]
```diff
@@ -394,11 +395,13 @@ def remove(self):
         else:
             logging.debug("Image file %s not found", self.image_filename)
 
-    def info(self, force_share=False):
+    def info(self, force_share=False, output="human"):
         """
         Run qemu-img info command on image file and return its output.
+
+        :param output: string of output format(`human`, `json`)
```
Comment (vivianQizhu) [inline virttest/qemu_storage.py]: Nice one.

File: virttest/qemu_storage.py (hunk 1 above)
Comment (vivianQizhu) [reply virttest/qemu_storage.py]: Should have a `:return` in the doc string.

Comment (hereischen) [reply virttest/qemu_storage.py]: updated, thanks

Comment (hereischen) [reply virttest/qemu_storage.py]: any other comments?

File: virttest/qemu_storage.py [hunk 3]
```diff
@@ -631,6 +634,25 @@ def amend(self, params, cache_mode=None, ignore_status=False):
...
 
+    def resize(self, size, shrink=False, ignore_status=False):
+        """
+        Qemu image resize wrapper.
+
+        :param size: string of size representations.(eg. +1G, -1k, 1T)
+        :param shrink: boolean
+        :param ignore_status: whether to raise an exception when command
+                              status code != 0 (False), or not (True)
+        :return: process.CmdResult object containing the result of the
```
Comment (luckyh) [inline virttest/qemu_storage.py]: Should be `string of the result of the command`.

**Pull request merged**